winpr/libwinpr/utils/asn1/asn1.c

```diff
@@ -1401,3 +1401,12 @@ size_t WinPrAsn1DecReadContextualSequence(WinPrAsn1Decoder* dec, WinPrAsn1_tagId
 	Stream_Seek(&dec->source, ret);
 	return ret;
 }
+
+void WinPrAsn1DecGetStream(WinPrAsn1Decoder* dec, wStream* s)
```

Comment:
Maybe change to `wStream WinPrAsn1DecGetStream(WinPrAsn1Decoder* dec)`

Reply:
On my side I like it this way.

Reply:
@hardening well, this is kind of a `allocator` function, so you can do `wStream s = WinPrAsn1DecGetStream(dec);` in a single line while preventing doing something like `wStream* s = NULL; WinPrAsn1DecGetStream(dec, s);`